backend/auth/models/user.go

```diff
@@ -3,6 +3,8 @@ package models
 type User struct {
 	Model
 
-	Email    string `gorm:"unique_index;not null"`
-	Password string
+	Email               string `gorm:"unique_index;not null"`
+	Password            string
+	RegistrationToken   string
+	AuthenticationToken string
```

Comment:
Shouldn't we store this in a separate table? There's a high chance people will want to sign in on multiple devices. We'll also need to store the `RefreshToken`, `DeviceName`and `ClientID`.
\+ I would stick with the oAuth2 naming conventions, and use `AccessToken`

Reply:
\+ need to have an expiry date

Reply:
Yes we should, this is only a temporary implementation. Main focus is GraphQL. This will go.